pkg/scheduler/partition_test.go

```diff
@@ -361,6 +361,35 @@ func TestRemoveNodeWithPlaceholders(t *testing.T) {
 	assert.Equal(t, 0, len(ph.Releases), "placeholder should have no releases linked anymore")
 }
 
+func TestCalculateNodesResourceUsage(t *testing.T) {
+	partition, err := newBasePartition()
+	assert.NilError(t, err, "partition create failed")
+	oldCapacity := resources.NewResourceFromMap(map[string]resources.Quantity{"first": 100})
+	node := newNodeMaxResource(nodeID1, oldCapacity)
+	err = partition.AddNode(node, nil)
+	assert.NilError(t, err)
+
+	occupiedResources := resources.NewResourceFromMap(map[string]resources.Quantity{"first": 50})
+	alloc := objects.NewAllocation(allocID, nodeID1, newAllocationAsk("key", "appID", occupiedResources))
+	node.AddAllocation(alloc)
+	usageMap := partition.calculateNodesResourceUsage()
+	assert.Assert(t, node.GetAvailableResource().Resources["first"] == 50)
```

Comment:
can we change this to an `assert.Equal`?
It will require replacing the 50 with `resources.Quantity(50)` i.e.:
```
assert.Equal(t, node.GetAvailableResource().Resources["first"], resources.Quantity(50))
```